plugins/gcloud/lib/samson_gcloud/image_tagger.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 module SamsonGcloud
   class ImageTagger
-    PRODUCTION_TAG = 'production'
```

Comment:
We should keep this so it can be used [here](https://github.com/zendesk/samson/pull/3319/files#diff-ee1c2f12150e834b6ae360c7421fb2a9R19)